exercises.cabal

```
@@ -69,12 +69,17 @@ library
                        Lecture2
                        Lecture3
                        Lecture4
+  build-depends:       base
+                     , split
+		     , bytestring
+		     , text
```

That's a very easy way to get performance but you can compiler your code with better optimizations 🙂 
```suggestion
		     , text
  ghc-options: -O2
```